pkg/cnab/extended_bundle.go

```diff
@@ -36,6 +40,24 @@ func LoadBundle(c *portercontext.Context, bundleFile string) (ExtendedBundle, er
 	return NewBundle(*bun), nil
 }
 
+func (b ExtendedBundle) Validate() error {
+	err := b.Bundle.Validate()
+	if err != nil {
+		return fmt.Errorf("invalid bundle: %w", err)
+	}
+
+	supported, err := semver.NewConstraint(SupportedVersion)
+	if err != nil {
+		return fmt.Errorf("invalid supported version %s: %w", SupportedVersion, err)
+	}
+	defaultVersion, err := semver.NewVersion(string(bundle.GetDefaultSchemaVersion()))
+	if err != nil {
+		return err
+	}
+	_, err = schema.ValidateSchemaVersion(schema.CheckStrategyExact, supported, string(b.SchemaVersion), defaultVersion)
```

Comment:
I thought we were going to use the configuration setting so that someone could use a looser check, or skip it entirely. Like we do when validating the manifest schema version?

https://github.com/getporter/porter/blob/07947b99479e50149e9ae898d4d2ca7cc119fb45/pkg/manifest/manifest.go#L1126-L1129

If we do make it configurable, you will want to handle any possible warnings returned (it's being ignored right now):

https://github.com/getporter/porter/blob/07947b99479e50149e9ae898d4d2ca7cc119fb45/pkg/manifest/manifest.go#L184-L190